### DISCUSSION TITLE
Getting start with reComputer J30/40 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1233] [Q&A]
# Getting start with reComputer J30/40 | Seeed Studio Wiki

Flash JetPack to reComputer J4012 (J401 carrier board)

https://wiki.seeedstudio.com/reComputer_J30_40_with_Jetson_getting_start/

**Comment by lucasrebra** [comment]
I bought the dev kit but I don't find if could connect to wifi. I think it doesn't have the M2 wifi module to do it

**Comment by Weavor74** [comment]
My J3011, will boot into desktop mode, only in ACPI selected in the bios Hardware description. When it does boot no keyboard or mouse is available.

**Reply by nfs0619** [reply]
Thanks for reaching out! We recommend contacting our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io). They will be able to provide you with more detailed guidance on your setup.

**Comment by ilhamrch** [comment]
Hello Nvidai Team

I have purchased Nvidia Jetson Nano Orin with J3010 series. However, when I did the settings for my needs in AI processing especially YOLO there was an problem. The problem is suddenly the Jetson Nano that I have cannot appear on the Monitor display that I connect. I have tried using DP, HDMI, and VGA. but all of that did not work. I have also checked the physical condition thoroughly and checked the HDMI Port condition also there is no problem. Do you have a solution to the problem on my Nvidia Jetson Nano Orin?

**Reply by nfs0619** [reply]
Based on what you’ve described, we suggest trying the following:

1. Try using a different monitor: Sometimes compatibility issues with specific monitors can cause the display problem. If you have access to another display, please try it out to see if the issue persists.

2. Reflash JetPack: If changing the monitor doesn’t help, we recommend reflashing your Jetson Nano Orin with the latest version of JetPack. This can sometimes resolve display-related issues caused by software or firmware inconsistencies.

If the issue still persists after trying both of these steps, please get in touch with our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)** for further assistance.

**Comment by blaisefinix** [comment]
I am having trouble installing OpenCV CUDA on a Jetson Nano 16GB supercomputer J4012. When I install it and follow the instructions available on various websites, I encounter this error:

"Reading package lists... Done Building dependency tree... Done Reading state information... Done cuda is already the newest version (12.6.11-1). The following packages were automatically installed and are no longer required: gdal-data libaec0 libarmadillo10 libarpack2 libavcodec-dev libavcodec58 libavdevice58 libavfilter7 libavformat-dev libavformat58 libavutil-dev libavutil56 libblosc1 libcfitsio9 libcharls2 libdc1394-dev libdeflate-dev libevent-core-2.1-7 libevent-pthreads-2.1-7 libexif-dev libfabric1 libfreexl1 libfyba0 libgdal30 libgdcm-dev libgdcm3.0 libgeos-c1v5 libgeos3.10.2 libgeotiff5 libgl2ps1.4 libglew2.2 libgphoto2-dev libhdf4-0-alt libhdf5-103-1 libhdf5-hl-100 libheif1 libhwloc-plugins libhwloc15 libilmbase-dev libjbig-dev libjpeg-dev libjpeg-turbo8-dev libjpeg8-dev libkmlbase1 libkmldom1 libkmlengine1 liblept5 libminizip1 libmysqlclient21 libnetcdf19 libodbc2 libodbcinst2 libogdi4.1 libopencv-calib3d4.5d libopencv-contrib4.5d libopencv-dnn4.5d libopencv-features2d4.5d libopencv-flann4.5d libopencv-highgui4.5d libopencv-imgcodecs4.5d libopencv-imgproc4.5d libopencv-ml4.5d libopencv-objdetect4.5d libopencv-photo4.5d libopencv-shape4.5d libopencv-stitching4.5d libopencv-superres4.5d libopencv-video4.5d libopencv-videoio4.5d libopencv-videostab4.5d libopencv-viz4.5d libopenexr-dev libopenmpi3 libpmix2 libpng-dev libpostproc55 libproj22 libraw1394-dev librttopo1 libsocket++1 libspatialite7 libssh-gcrypt-4 libsuperlu5 libswresample-dev libswresample3 libswscale-dev libswscale5 libsz2 libtbb-dev libtesseract4 libtiff-dev libtiffxx5 libucx0 liburiparser1 libvdpau1 libvtk9.1 libxerces-c3.2 libxnvctrl0 mysql-common proj-data unixodbc-common Use 'sudo apt autoremove' to remove them. 0 upgraded, 0 newly installed, 0 to remove and 7 not upgraded. 8 not fully installed or removed. After this operation, 0 B of additional disk space will be used. Setting up nvidia-l4t-bootloader (36.4.7-20250918154033) ... Warning: Cannot get compatible board name. 3767-000-0000--1--recomputer-orin-super-j401- TNSPEC 3767-301-0000-G.1-1-0-recomputer-orin-super-j401- COMPATIBLE_SPEC 3767-000-0000--1--recomputer-orin-super-j401- TEGRA_BOOT_STORAGE nvme0n1 TEGRA_CHIPID 0x23 TEGRA_OTA_BOOT_DEVICE /dev/mtdblock0 TEGRA_OTA_GPT_DEVICE /dev/mtdblock0 Info: Write TegraPlatformCompatSpec with 3767-000-0000--1--recomputer-orin-super-j401-. Info: The esp is already mounted to /boot/efi. INFO. Dump slots info: Current version: 36.4.3 Capsule update status: 0 Current bootloader slot: A Active bootloader slot: A num_slots: 2 slot: 0, status: normal slot: 1, status: normal INFO. Dump nv_boot_control.conf: TNSPEC 3767-301-0000-G.1-1-0-recomputer-orin-super-j401- COMPATIBLE_SPEC 3767-000-0000--1--recomputer-orin-super-j401- TEGRA_BOOT_STORAGE nvme0n1 TEGRA_CHIPID 0x23 TEGRA_OTA_BOOT_DEVICE /dev/mtdblock0 TEGRA_OTA_GPT_DEVICE /dev/mtdblock0 ERROR. 3767-000-0000--1--recomputer-orin-super-j401- does not match any known boards. dpkg: error processing package nvidia-l4t-bootloader (--configure): installed nvidia-l4t-bootloader package post-installation script subprocess returned error exit status 1 Setting up nvidia-l4t-kernel (5.15.148-tegra-36.4.7-20250918154033) ... Using the existing boot entry 'primary' Warning: Cannot get compatible board name. 3767-000-0000--1--recomputer-orin-super-j401- TNSPEC 3767-301-0000-G.1-1-0-recomputer-orin-super-j401- COMPATIBLE_SPEC 3767-000-0000--1--recomputer-orin-super-j401- TEGRA_BOOT_STORAGE nvme0n1 TEGRA_CHIPID 0x23 TEGRA_OTA_BOOT_DEVICE /dev/mtdblock0 TEGRA_OTA_GPT_DEVICE /dev/mtdblock0 Info: Write TegraPlatformCompatSpec with 3767-000-0000--1--recomputer-orin-super-j401-. Info: The esp is already mounted to /boot/efi. Starting kernel post-install procedure. Rootfs AB is not enabled. ERROR. Procedure for A_kernel update FAILED. Cannot install package. Exiting... dpkg: error processing package nvidia-l4t-kernel (--configure): installed nvidia-l4t-kernel package post-installation script subprocess returned error exit status 1 dpkg: dependency problems prevent configuration of nvidia-l4t-kernel-headers: nvidia-l4t-kernel-headers depends on nvidia-l4t-kernel (= 5.15.148-tegra-36.4.7-20250918154033); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-kernel-headers (--configure): dependency problems - leaving unconfigured dpkg: dependency problems prevent configuration of nvidia-l4t-jetson-io: nvidia-l4t-jetson-io depends on nvidia-l4t-kernel (>> 5.15.148-tegra-36.4-0); however: Package nvidia-l4t-kernel is not configured yet. nvidia-l4t-jetson-io depends on nvidia-l4t-kernel (<< 5.15.148-tegra-36.5-0); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-jetson-io (--configure): dependency problems - leaving unconfigured No apport report written because the error message indicates its a followup error from a previous failure. No apport report written because MaxReports is reached already No apport report written because MaxReports is reached already dpkg: dependency problems prevent configuration of nvidia-l4t-kernel-oot-modules: nvidia-l4t-kernel-oot-modules depends on nvidia-l4t-kernel (= 5.15.148-tegra-36.4.7-20250918154033); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-kernel-oot-modules (--configure): dependency problems - leaving unconfigured dpkg: dependency problems prevent configuration of nvidia-l4t-display-kernel: nvidia-l4t-display-kernel depends on nvidia-l4t-kernel (= 5.15.148-tegra-36.4.7-20250918154033); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-display-kernel (--configure): dependency problems - leaving unconfigured dpkg: dependency problems prevent configuration of nvidia-l4t-kernel-oot-headers: nvidia-l4t-kernel-oot-headers depends on nvidia-l4t-kernel (= 5.15.148-tegra-36.4.7-20250918154033); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-kerNo apport report written because MaxReports is reached already No apport report written because MaxReports is reached already No apport report written because MaxReports is reached already nel-oot-headers (--configure): dependency problems - leaving unconfigured dpkg: dependency problems prevent configuration of nvidia-l4t-kernel-dtbs: nvidia-l4t-kernel-dtbs depends on nvidia-l4t-kernel (= 5.15.148-tegra-36.4.7-20250918154033); however: Package nvidia-l4t-kernel is not configured yet. dpkg: error processing package nvidia-l4t-kernel-dtbs (--configure): dependency problems - leaving unconfigured Processing triggers for nvidia-l4t-initrd (36.4.7-20250918154033) ... Include /etc/nv-update-initrd/list.d/binlist Include /etc/nv-update-initrd/list.d/modules nv-update-initrd: Updating /boot/initrd from /etc/nv-update-initrd/list.d for kernel version 5.15.148-tegra.. Add /usr/sbin/nvluks-srv-app Add /lib/modules/5.15.148-tegra/updates/drivers/net/ethernet/nvidia/nvethernet/nvethernet.ko Add /lib/modules/5.15.148-tegra/updates/drivers/nvpps/nvpps.ko Add /lib/modules/5.15.148-tegra/updates/drivers/net/ethernet/realtek/r8126/r8126.ko Add /lib/modules/5.15.148-tegra/updates/drivers/net/ethernet/realtek/r8168/r8168.ko Add /lib/modules/5.15.148-tegra/modules* Add /lib/modules/5.15.148-tegra/kernel/drivers/nvme/host/nvme.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/nvme/host/nvme-core.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/thermal/tegra/tegra-bpmp-thermal.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/pwm/pwm-tegra.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/hwmon/pwm-fan.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/pci/controller/dwc/pcie-tegra194.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/phy/tegra/phy-tegra194-p2u.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/usb/gadget/udc/tegra-xudc.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/usb/typec/ucsi/typec_ucsi.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/usb/typec/ucsi/ucsi_ccg.ko Add /lib/modules/5.15.148-tegra/kernel/drivers/usb/typec/typec.ko Updating modprobe.d configuration directories for modprobe.. Add config /etc/modprobe.d/alsa-base.conf Add config /etc/modprobe.d/bcmdhd.conf Add config /etc/modprobe.d/blacklist-ath_pci.conf Add config /etc/modprobe.d/blacklist.conf Add config /etc/modprobe.d/blacklist-firewire.conf Add config /etc/modprobe.d/blacklist-framebuffer.conf Add config /etc/modprobe.d/blacklist-modem.conf Add config-link /etc/modprobe.d/blacklist-oss.conf Add config /etc/modprobe.d/blacklist-rare-network.conf Add config /etc/modprobe.d/deepin-screen-recorder.conf Add config /etc/modprobe.d/denylist-nouveau.conf Add config /etc/modprobe.d/denylist-oot-modules-audio.conf Add config /etc/modprobe.d/denylist-ramoops.conf Add config /etc/modprobe.d/denylist-tegra-safety.conf Add config /etc/modprobe.d/denylist-tpm-ftpm-tee.conf Add config /etc/modprobe.d/iwlwifi.conf Add config /etc/modprobe.d/nvgpu.conf Add config /etc/modprobe.d/nvidia-display.conf Add config /etc/modprobe.d/tegra-udrm.conf Add config /lib/modprobe.d/aliases.conf Add config /lib/modprobe.d/fbdev-blacklist.conf Add config /lib/modprobe.d/systemd.conf Cleaning up the temporary directory for updating the initrd.. Errors were encountered while processing: nvidia-l4t-bootloader nvidia-l4t-kernel nvidia-l4t-kernel-headers nvidia-l4t-jetson-io nvidia-l4t-kernel-oot-modules nvidia-l4t-display-kernel nvidia-l4t-kernel-oot-headers nvidia-l4t-kernel-dtbs E: Sub-process /usr/bin/dpkg returned an error code (1)"

Perhaps you have a solution related to the guide for installing OpenCV CUDA on a Super J4012 recomputer with Jetpack 6.2 pre-installed
Please provide guidance as I want to run an object detection-based program using YOLO. This requires CUDA processing to avoid excessive delay.
I greatly appreciate any answers you can provide.

**Reply by bleaaach** [reply]
Hi, you can flash our official image first, then execute the following two commands:
`sudo apt update`
`sudo apt install nvidia-jetpack`
This will install SDKs such as TensorRT, OpenCV, and CUDA together.

**Reply by kyuhyong** [reply]
J9, J12 connectors only have 1x lane SCI0 and CSI2 while devkit comes with x2 lane CSI on J20 (CSI0 and 1), J21 (CSI2 and 3). 
How can I connect CSI1 or CSI3 on J3010 board?

**Comment by kyuhyong** [comment]
J9, J12 connectors only have 1x lane SCI0 and CSI2 while devkit comes with x2 lane CSI on J20 (CSI0 and 1), J21 (CSI2 and 3).
How can I connect CSI1 or CSI3 on J3010 board?

**Reply by bleaaach** [reply]
Hi, CSI1/CSI3 cannot be directly connected to the J401 motherboard as they are not exposed at the hardware level.

**Comment by glpuga** [comment]
Hi! can the usb-c port be configured into host-mode? Thanks!

**Reply by bleaaach** [reply]
Hi. This USB-C port cannot be set to Host Mode. It is primarily intended for flashing firmware (e.g., JetPack image flashing via a USB connection to a PC) or to function as a peripheral device recognized by a host, rather than connecting USB peripherals such as USB flash drives and keyboards.

**Reply by glpuga** [reply]
Thanks for the answer!
